notesapi/v1/models.py

```diff
@@ -54,4 +57,5 @@ def as_dict(self):
             'ranges': json.loads(self.ranges),
             'created': created,
             'updated': updated,
+            'tags': json.loads(self.tags) if self.tags else []
```

Comment:
Yes, I was. I wanted to make sure the API is robust to whether or not notes are present (whether or not the plugin is being used).